ucp/useraccounts_module.php

```diff
@@ -464,15 +417,19 @@ function main($id, $mode)
 						trigger_error($this->language->lang('FORM_INVALID') . '<br><br>' . $this->user->lang('RETURN_UCP', '<a href="' . $this->u_action . '">', '</a>'));
 					}
 
-					// Get current item order value
-					$sql = "SELECT badge_order as current_order
-						FROM $user_badge_table
-						WHERE item_id = $item_id"; // No need to check for user_id, item_id is the table's primary key.
+					// Get current item order value.
+					$sql_array = array(
+						'item_id' => $item_id
+					);
+
+					$sql = 'SELECT badge_order as current_order
+						FROM ' . $user_badge_table . '
+						WHERE ' . $this->db->sql_build_array('SELECT', $sql_array); // No need to check for user_id, item_id is the table's primary key.
```

Comment:
Using `sql_build_array` here hurts readability without any real benefit.

Reply:
`item_id` is in this case user input but it is already casted to an integer so I think it is safe to directly use the var.